signer/src/context/signer_state.rs

```diff
@@ -174,6 +182,8 @@ impl SbtcLimits {
             Some(Amount::MAX_MONEY),
             Some(Amount::ZERO),
             Some(Amount::ZERO),
+            Some(u64::MAX),
```

Comment:
The `u64::MAX` scares me a bit (assuming a certain usage within queries), what about using `1` instead? It should have the same effect

Reply:
oh right, if we don't have withdrawals then it would reset. Let's keep it as is, we can re-asses later it this cause issues with the query.